package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Header Bidding Management Library",
   "main": "src/prebid.js",
   "scripts": {
-    "test": "gulp test && gulp mocha"
+    "test": "gulp test && gulp mocha",
```

Comment:
No having these mapped to npm task is fine, I'm just pointing out that `gulp test` will run `gulp test && gulp mocha`